ttnn/ttnn/operations/core.py

```diff
@@ -650,7 +650,7 @@ def to_layout(tensor, layout: ttnn.Layout, dtype: ttnn.DataType = None):
         raise RuntimeError(f"Unsupported layout conversion from {tensor.layout} to {layout}")
 
     is_on_device = ttnn.is_tensor_storage_on_device(tensor)
-    if is_on_device and tensor.dtype not in {ttnn.bfloat16, ttnn.bfloat8_b, ttnn.bfloat4_b}:
+    if is_on_device and tensor.dtype not in {ttnn.bfloat16, ttnn.bfloat8_b, ttnn.bfloat4_b, ttnn.float32, ttnn.uint32}:
```

Comment:
thanks for updating this... uint16 and int32 should also be added here too, right?

Reply:
yes, I will update uint16, but int32 isn't exposed in the ttnn yet so I can't add it. 
I do plan to get to it so I will add it then